cmd/collectors/zapi/plugins/shelf/shelf.go

```diff
@@ -184,6 +184,8 @@ func (my *Shelf) Run(data *matrix.Matrix) ([]*matrix.Matrix, error) {
 			my.Logger.Debug().Msgf("fetching %d [%s] instances", len(objectElem.GetChildren()), attribute)
 
 			for _, obj := range objectElem.GetChildren() {
+				// To decide instance would be available in metric or not
+				isInclude := true
```

Comment:
minor: but can you move this down closer to where it's used, ~ line 199

Reply:
done.